haskell/haskell.bzl

```diff
@@ -136,7 +136,7 @@ def _mk_binary_rule(**kwargs):
       main_file = attr.label(
         allow_single_file = FileType([".hs", ".hsc", ".lhs"]),
         doc = "File containing `Main` module.",
-      )
+      ),
```

Comment:
The same here, this is not necessary (and extra newline below).

Reply:
Done